Day-I-Social-Card/social-card/src/components/FooterComponents/Footer.css

```diff
@@ -0,0 +1,24 @@
+.footer-container {
+  position: relative;
+  margin-top: 30px;
+  margin: 15px;
+  left: 80px;
+  display: flex;
+}
+.fake {
+}
+#comment {
```

Comment:
I would try to avoid using ID's if possible. It's almost always better to just have className instead that is really specific, because IDs are so heavy already.